### DISCUSSION TITLE
Pull request checklist template

### DISCUSSION THREAD

**Discussion by Avi197** [explosion/spaCy#9409] [Help: Other Questions] [answered]
Hi, I can't seem to find the checklist template for pull request, and also can't find the Spacy contributor agreement. 

Would be lovely if someone show me the link, thanks

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @Avi197 , here are the links for the [PR template](https://github.com/explosion/spaCy/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and [contributor agreement](https://github.com/explosion/spaCy/blob/master/.github/CONTRIBUTOR_AGREEMENT.md). Curious as to why you need this? the PR template should show up by the time you make the Pull Request

**Reply by Avi197** [reply]
Oh thanks a lot, somehow when I create a pull request, the PR template didn't show up for me
Also I was following [this PR](https://github.com/explosion/spaCy/pull/2155) , as it's the only one contribution for Vietnamese Spacy, and the check list look a bit different from the recent one

**Reply by polm** [reply]
For anyone seeing this later - note that the PR checklist has changed over time. Just use whatever shows up when you create a PR, even if it's different from other PRs you've seen before. If for some technical reason the checklist doesn't show up at all that's fine, we'll ask you about it in Discussion.
